pom.xml

```diff
@@ -15,6 +15,12 @@
     </properties>
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.json/json -->
+        <dependency>
+            <groupId>org.json</groupId>
```

Comment:
org.json.json is already included in pom.xml in the current develop branch => please update your PR.

Reply:
yes fixed!